modules/data_access/get_named_location_geolocations.py

```diff
@@ -74,6 +77,15 @@ def get_named_location_geolocations(connector: DbConnector, named_location_id: i
                 # build the reference feature
                 reference_location_properties = dict(name=named_location_offset_name, locations=reference_locations)
                 reference_feature = Feature(geometry=None, properties=reference_location_properties)
+            else:
```

Comment:
The site level location data cannot be located within the reference location chain unless the site is truly a reference location. This will break geolocation computations that rely on the reference location chain. We can include site-level metadata in the geolocation file, but it needs to go somewhere else. Let's put it in it's own "site_location" field at the same location in the json hierarchy as the new asset information. This will also allow us to easily filter it for products in which we don't need it in processing.

Reply:
I updated the code using site_location field. Attached is an example of the new output file 
[dualfan_341_locations.json](https://github.com/user-attachments/files/19373108/dualfan_341_locations.json)